build-tools/packages/build-cli/docs/generate.md

```diff
@@ -246,6 +247,57 @@ DESCRIPTION
 
 _See code: [src/commands/generate/entrypoints.ts](https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/commands/generate/entrypoints.ts)_
 
+## `flub generate typetests`
+
+Generates type tests for a package or group of packages.
+
+```
+USAGE
+  $ flub generate typetests [-v | --quiet] [--level public|alpha|beta|internal|legacy] [--publicFallback] [--concurrency
+    <value>] [--all | --dir <value> | --packages | -g client|server|azure|build-tools|gitrest|historian|all |
+    --releaseGroupRoot client|server|azure|build-tools|gitrest|historian|all] [--private] [--scope <value> | --skipScope
+    <value>]
+
+FLAGS
+  --concurrency=<value>  [default: 25] The number of tasks to execute concurrently.
+  --level=<option>       [default: internal] What API level to generate tests for.
+                         <options: public|alpha|beta|internal|legacy>
+  --publicFallback       Use the public entrypoint as a fallback if the API at the requested level is not found. Pass
+                         --no-publicFallback to disable.
+
+PACKAGE SELECTION FLAGS
+  -g, --releaseGroup=<option>...      Run on all child packages within the specified release groups. This does not
+                                      include release group root packages. To include those, use the --releaseGroupRoot
+                                      argument. Cannot be used with --all, --dir, or --packages.
+                                      <options: client|server|azure|build-tools|gitrest|historian|all>
+      --all                           Run on all packages and release groups. Cannot be used with --all, --dir,
+                                      --releaseGroup, or --releaseGroupRoot.
+      --dir=<value>                   Run on the package in this directory. Cannot be used with --all, --dir,
```

Comment:
In practice we will usually use the --dir option and run it per-package just like we do with fluid-type-test-generator today.